.github/workflows/tag.yml

```diff
@@ -0,0 +1,60 @@
+name: Earthly +tag
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+env:
+  FORCE_COLOR: 1
+  CARGO_TERM_COLOR: always
+  NIGHTLY_TOOLCHAIN: nightly
+
+jobs:
+  tag:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    env:
+      FORCE_COLOR: 1
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+
+    steps:
+      # Setup repo and add caching
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+
+      #Deps
+      - name: Set up Earthly
+        uses: earthly/actions-setup@v1
+        with:
+          use-cache: true
```

Comment:
```suggestion
        with:
          version: v0.8.0
          use-cache: true
```

Reply:
do we want to lock to 0.8 or is latest ?

Reply:
Oh if we leave it out it will be latest? I think that's a good idea